api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
> Also, do we need to return a length of the buffer if we have the buffer?

In my implementation we return a unsafe.Pointer, not a buffer, to give the user more flexibility and let decide what to do with this raw pointer. unsafe.Pointer doesn't have length.

> I think we shouldn't return it b/c we have the value which could be measured.

I don't understand what exactly do you mean. If you mean that length contained in byte slice than we don't return the byte slice now, just a raw pointer, just as C.
Should we return byte slice? And force user to use byte slice (It seems ok to me because go is a high level language and as far as I can see in std go library returning unsafe.Pointer not too often)? Or should we map C API 1 on 1 and pass a int pointer to obtain buffer length? Or get buffer length as a return value from function?

> @pytlesk4, besides, we don't have to increase the version b/c previous implementation was incorrect. So, it's just a butfix.

Yeah, it seems to me as a bugfix and I think it should be merged in all versions, but it's up to you.